runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -316,31 +396,40 @@ public CorfuRuntime registerBeforeRpcHandler(Runnable handler) {
     @Getter
     @Setter
     public Function<String, IClientRouter> getRouterFunction = overrideGetRouterFunction != null
-            ? (address) -> overrideGetRouterFunction.apply(this, address) : (address) -> {
-
-                // Return an existing router if we already have one.
-                if (nodeRouters.containsKey(address)) {
-                    return nodeRouters.get(address);
-                }
-
-                NodeLocator node = NodeLocator.parseString(address);
-                // Generate a new router, start it and add it to the table.
-                NettyClientRouter router = new NettyClientRouter(node, getParameters());
-                log.debug("Connecting to new router {}", node);
-                try {
-                    router.addClient(new LayoutClient())
-                            .addClient(new SequencerClient())
-                            .addClient(new LogUnitClient().setMetricRegistry(metrics != null
-                                            ? metrics : CorfuRuntime.getDefaultMetrics()))
-                            .addClient(new ManagementClient())
-                            .start();
-                    nodeRouters.put(address, router);
-                } catch (Exception e) {
-                    log.warn("Error connecting to router", e);
-                }
-                return router;
-            };
+            ? (address) -> overrideGetRouterFunction.apply(this, address) : (address) ->
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "address" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the parentheses around the "address" parameter [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1611)